.github/workflows/test.yml

```
@@ -5,7 +5,7 @@ on: [push, pull_request]
 
 env:
     PROJECT: 'SmartDeviceLink-iOS.xcodeproj'
-    DESTINATION: 'platform=iOS Simulator,name=iPhone 11,OS=14.1'
+    DESTINATION: 'platform=iOS Simulator,name=iPhone 11,OS=14.4'
```

```suggestion
    DESTINATION: 'platform=iOS Simulator,name=iPhone 12,OS=14.4'
```

We'll have to update less often and we don't have the snapshot tests to deal with now.